readme.md

```diff
@@ -1,115 +1,96 @@
+For 2019: 
+1. Run `cd 2019/python` to navigate to the python directory. 
+2. Run `python3 generate_2019_error_files.py`
+3. The error files files for testing syntax, validity, and quality edit test files should appear in the following diretories:
+	- Syntax: `2019/edits_files/test_files/{Bank Name}/syntax`
+	- Validity: `2019/edits_files/test_files/{Bank Name}/validity`
+	- Quality: `2019/edits_files/test_files/{Bank Name}/quality`
 
-These functions include:
-- `write_file(path, ts_input, lar_input, name="test_file.txt")` 
-	Writes TS and LAR dataframes to a HMDA submission test file with a specified file path and directory. 
+Note: the filepath and filename for test files can be adjusted in the [2019 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml) YAML configuration file under the keys, `syntax_filepath`, `validity_filepath`, and `quality_filepath`.  
 
-- `read_data_frames(ts_df, lar_df)` 
-	Reads in a HMDA submission test file and outputs a TS and LAR dataframe. 
+For 2018: 
+1. Run `cd 2018/python` to navigate to the python directory. 
+2. Run `python3 generate_2018_error_files.py`
+3. The error files files for testing syntax, validity, and quality edit test files should appear in the following diretories:
+	- Syntax: `2018/edits_files/test_files/{Bank Name}/syntax`
+	- Validity: `2018/edits_files/test_files/{Bank Name}/validity`
+	- Quality: `2018/edits_files/test_files/{Bank Name}/quality`
 
-- `unique_uli(new_lar_df, lei)`
-	Writes a set of unique ULI’s with check digits to a LAR dataframe given an LEI input. 
+Note: the filepath and filename for test files can be adjusted in the [2018 Test Filepaths](https://github.com/cfpb/hmda-test-files/tree/master/2018/configurations/test_filepaths.yaml) YAML configuration file under the keys, `syntax_filepath`, `validity_filepath`, and `quality_filepath`. 
 
-- `new_lar_rows(row_count, lar_df, ts_df)` 
-	Duplicates or retracts rows in a LAR dataframe to produce a file with a certain number of rows. Provides capabilities for creating LAR dataframes with large row counts. 
+## Generating Large Files 
+Often, large files would need to be generated to support load testing. The repository allows users to creates a test file based on existing test file data to create a new file with a larger number of rows.
 
-- `row_by_row_modification(lar_df, yaml_filepath='configurations/row_by_row_modification.yaml')`
-	Changes a LAR dataframe given changes specified in a yaml file. It allows users to test for minor changes in LAR data.
+To generate large files for 2019: 
+1. Run `cd 2019/python` to navigate to the python directory.
+2. Adjust the [2019 File Large File Script Configuration](https://github.com/cfpb/hmda-test-files/tree/master/2019/configurations/test_filepaths.yaml) to specify bank name, lei, tax id, row count, output filepath, and output filename. 
```

Comment:
Defaults should work for a stated use case. Please list the default parameters, provide the link and state that users can set them to what they need.

See comments above on similar steps.

Reply:
Sounds good - the default configurations should be the same for each year's file, and I am adding this to the discussion on configurations in the first paragraph of this section.